### ISSUE TITLE
Android - Stuck in waiting room

### ISSUE THREAD

**Issue by bryantmh** [byucs340ta/Spring2019#4]
Can't leave a game waiting room without restarting the app

**Comment by bkleinman7** [comment]
Is this a bug? Can you specify where it talks about this?

**Comment by bryantmh** [comment]
You guys submitted the same bug for our project, so....

**Comment by Bobleyl** [comment]
This is different because the user is stuck in the waiting room and can't go back or start a game either.  Ours, you can still start a game when the waiting room fills up.

**Comment by bryantmh** [comment]
Yeah, that's how ours works. You can start if you have at least 2 players. If you wanna leave, refresh the page.

**Comment by bryantmh** [comment]
Literally the same, only it's a royal pain to restart the android app.

**Comment by Bobleyl** [comment]
I'm not sure if yall know this, but if any of the players leave the waiting room, nobody can start the game, even if it fills up or has enough players.

**Comment by Bobleyl** [comment]
That's the bug I was originally reporting.
